package.json

```diff
@@ -61,7 +61,7 @@
   },
   "homepage": "https://github.com/argentlabs/argent-contracts#readme",
   "dependencies": {
-    "@openzeppelin/contracts": "3.0.1",
+    "@openzeppelin/contracts": "3.4.0",
```

Comment:
We can make better use of the `SafeMath` library and use their SafeCast methods introduced since [v3.1](https://github.com/OpenZeppelin/openzeppelin-contracts/releases/tag/v3.1.0). [`toUint128`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/SafeCast.sol#L31), `toUint64` and [`min`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/math/Math.sol#L19) functions. Which are exact replicas of what we have in `Utils` lib.